source/upgrade/important-upgrade-notes.rst

```diff
@@ -19,15 +19,39 @@ Important Upgrade Notes
+| v7.1                                               | A new configuration option ``MaxImageDecoderConcurrency`` indicates how many images can be decoded concurrently at once. The default is -1, and the value        |
+|                                                    | indicates the number of CPUs present. This affects the total memory consumption of the server. The maximum memory of a single image is dictated by               |
+|                                                    | ``MaxImageResolution * 24 bytes``. Therefore, we recommend that ``MaxImageResolution * MaxImageDecoderConcurrency * 24`` should be less than the allocated       |
+|                                                    | memory for image decoding.                                                                                                                                       |
+|                                                    +------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+|                                                    | New schema changes were introduced in the form of a new column, and its index. The following summarizes the test results measuring how long it took for the      |
+|                                                    | database queries to run with these schema changes:                                                                                                               |                                                                                                          
+|                                                    |                                                                                                                                                                  |
+|                                                    | - MySQL 12M Posts, 2.5M Reactions - ~1min 34s (Instance: PC with 8 cores, 16GB RAM)                                                                              |
+|                                                    | - PostgreSQL 12M Posts, 2.5M Reactions - ~1min 18s (Instance: db.r5.2xlarge)                                                                                     |
+|                                                    |                                                                                                                                                                  |
+|                                                    | For customers wanting an upgrade with substantially reduced downtime (~1/5th), they are encouraged to run the following queries before upgrading.                |
+|                                                    | This is fully backwards compatible but will obtain a lock on ``Reactions`` table, and users' reactions in the interval wouldn't be reflected in the database.    |                      
```

Comment:
This is incorrect. More discussion here: https://github.com/mattermost/docs/pull/5773. We should match whatever text is written there.